qemu/tests/slof_device_tree.py

```diff
@@ -53,14 +54,16 @@ def check_nonexist_aliases(vm_session):
     timeout = int(params.get("login_timeout", 600))
 
     session = vm.wait_for_login(timeout=timeout)
+    match_str = params['match_str']
```

Comment:
Move this line to `def get_info()` maybe better. It's up to you.
Acked-by: Yihuang Yu <yihyu@redhat.com>

Reply:
```
import time

d = {'name': 'xian'}
t = d['name']

start = time.time()
for i in range(100000000):
    v = t
print('elapse time %s.', start - time.time())


start = time.time()
for i in range(100000000):
    v = d['name']
print('elapse time %s.', start - time.time())

```